plugins/msol/msol.py

```diff
@@ -5,7 +5,7 @@ def msol_authenticate(url, username, password, useragent, pluginargs):
 
     data_response = {
         'result': None,    # Can be "success", "failure" or "potential"
-        'error': False,
+		'error' : False,
```

Comment:
Is it just me or are some of these randomly differently tabbed? Unsure how much that matters